File: Python/Ram.py
from Komponen import Komponen

class Ram(Komponen):
    def __init__(self, nama, kapasitas, tipe):
        super().__init__(nama)
        self.kapasitas = kapasitas
        self.tipe = tipe
    
    def get_info(self):
        return f"{self.nama} ({self.kapasitas} GB) {self.tipe}"

File: CPP/Ram.cpp
#include "Komponen.cpp"

class Ram : public Komponen {
private:
    int kapasitas;
    string tipe;
public:
    Ram() {}
    Ram(string nama, int kapasitas, string tipe) : Komponen(nama) {
        this->kapasitas = kapasitas;
        this->tipe = tipe;
    }
    void tampil() {
        cout << "Ram       : " << nama << " (" << kapasitas << " GB) " << tipe << endl;
    }
};